### DISCUSSION TITLE
"GitHub Galaxy Brain"

### DISCUSSION THREAD

**Discussion by NahuelOrselli** [drknzz/GitHub-Achievements#107] [Q&A] [answered]
Reply

**Comment by tonngw** [comment, marked as answer]
Can you mark my reply? I will mark it for you in https://github.com/drknzz/GitHub-Achievements/discussions/108

**Reply by tonngw** [reply]
> Reply

Can you mark my reply again, I haven't get it.

**Comment by tonngw** [comment]
help me mark it please, thanks

**Reply by NahuelOrselli** [reply]
i do another, reply in #109

**Reply by tonngw** [reply]
> i do another, reply in #109 

ok, i have replyed.
